docs/pages/sdk/snippets/preference-parameters.mdx

```diff
@@ -0,0 +1,63 @@
+
+### preference (optional)
+
+- Type `Preference`
+
+```ts twoslash
+type Attribution = {
+    auto: boolean;
+    dataSuffix?: never;
+} | {
+    auto?: never;
+    dataSuffix: `0x${string}`;
+}
+
+type Preference = {
+    options?: 'all' | 'smartWalletOnly' | 'eoaOnly';
+    attribution?: Attribution;
+}
+```
+
+#### `preference.options` (optional)
+- Type: `'all' | 'smartWalletOnly' | 'eoaOnly'`
+
+Determines which connection options users will see. Defaults to `all`.
+
+##### `all`
+Users will see Smart Wallet and mobile app connection options.
```

Comment:
just nitpicking, wondering if we could make the all, smartWalletOnly section smaller or indented, so the hierarchy is clearer

![image](https://github.com/user-attachments/assets/699137c5-6fd2-46f5-99a2-b5614b4a62eb)

Reply:
updated to use groupings

![image](https://github.com/user-attachments/assets/8eb32723-9400-4ec4-87ad-e85c17a0a632)